library/std/src/path.rs

```diff
@@ -2705,6 +2705,9 @@ impl Path {
 
     /// Returns `true` if the path points at an existing entity.
     ///
+    /// Warning: this method may be error-prone, consider using [`try_exists()`] instead!
+    /// It also has a risk of introducing time-of-check to time-to-use (TOCTOU) bugs.
```

Comment:
Shouldn't this be 

```suggestion
    /// It also has a risk of introducing time-of-check to time-of-use (TOCTOU) bugs.
```

to be consistent with the rest?

Reply:
Sure, thanks!